clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -523,6 +523,16 @@
             "NATIVE - Use platform binary"
           ],
           "description": "COBOL LS server runtime."
+        },
+        "cobol-lsp.formatting": {
+          "type": "string",
+          "default": "Uppercase",
```

Comment:
Default should be "None" for backward compatibility.

Reply:
Make sense, fixed